.eslintignore

```diff
@@ -5,3 +5,4 @@ coverage/
 **/*.d.ts
 /docs/_preview
 /docs/_loopback.io
+**/.eslintrc.js
```

Comment:
This is a bad idea. We are running Prettier as part of `eslint`, this line disables prettier formatting checks for `.eslintrc` files.